### DISCUSSION TITLE
Anonymous children vaccination

### DISCUSSION THREAD

**Discussion by NaveedAhmadHematmal** [VDTS/CodeX.VaxineSolution#39] [Ideas]
Children that don't belong to the team's area and don't have any registration should be vaccinated anonymously.
Because they are guests or children that are coming from other countries or other provinces. So we want to add a dedicated page to anonymous children in our current app workflow. Which place is a good fit for that page?

**Comment by SaeedaRasuly** [comment]
Well, I think it would be good if we add another tab by the name of Anonymous Children just as the Masjeed, influencer tabs

**Reply by NaveedAhmadHematmal** [reply]
Do you mean we should add one tab for anonymous to the Home page? Good idea.

**Reply by SaeedaRasuly** [reply]
Yes we can add one tab to home page.
